tests/cli-test.js

```diff
@@ -180,6 +180,38 @@ describe('CLI', function () {
     });
   });
 
+  describe(`--ignore-dependencies`, function () {
+    it('check console log', async function () {
+      const child = await runSupportedCmd([
+        `${__dirname}/fixtures/supported-project`,
+        `--ignored-dependencies @stefanpenner/a`,
+        `-i rsvp`,
+      ]);
+
+      expect(child).to.exitGracefully();
+      expect(child.stderr).to.includes(`Ignored: 2`);
```

Comment:
I would prefer just not mentioning how many ignored dependencies, combined with this suggestion to always suggest running with --verbose:
https://github.com/stefanpenner/supported/issues/18

Reply:
We should mention this to allow consistency with the number of total dependencies and number verified + number of remaining.
We have a running counter which shows the status and final status as well.
This is how it will look if we do not show ignored count. Where looks more like an issue as the count do not match the total.
```
Total Dependencies: 36, Verified: 35, Remaining: 0
```
if we add ignored count. it makes it complete.  
```
Total Dependencies: 36, Verified: 35, Remaining: 0, Ignored: 1
```

Reply:
ah gotcha, makes sense